servers/quarkus-config/src/main/java/org/projectnessie/quarkus/config/QuarkusStoreConfig.java

```diff
@@ -124,8 +126,20 @@ public interface QuarkusStoreConfig extends StoreConfig {
   @WithName(CONFIG_CACHE_CAPACITY_MB)
   OptionalInt cacheCapacityMB();
 
+  /**
+   * Nessie keeps so called soft-references of the cached Java objects in addition to the serialized
+   * representation around.
+   *
+   * <p>This toggle optionally enables this behavior.
+   */
+  @WithName(CONFIG_CACHE_ENABLE_SOFT_REFERENCES)
+  @WithDefault("" + DEFAULT_CONFIG_CACHE_ENABLE_SOFT_REFERENCES)
+  Optional<Boolean> cacheEnableSoftReferences();
```

Comment:
nit: why double default? here and in `PersistProvider`?

Reply:
that default is documented ;)